commands/serve/web/hot.js

```diff
@@ -62,4 +55,13 @@ export default function initiateWatch(info) {
     };
   }
   update();
+  if (info.types) {
+    const proms = info.types.map((t) => {
+      return getModule(t.name, t.url).then((mo) => {
+        window[t.name] = mo;
+      });
+    });
+    return Promise.all(proms);
+  }
+  return Promise.resolve();
```

Comment:
[optional]
alternative you could set window[t.name] to a promise
```
window[t.name] = getModule(t.name, t.url);
```
then you would not need to wait here
and I think that would be closer to how it work in a real mashup